DOCS-12051-Renew certificates for servers in a cluster or group (#543)
mulesoft/docs-runtime-manager: modules/ROOT/pages/servers-cert-renewal.adoc

The status for the server must be `Running` to update its certificate.

> To renew a certificate for a standalone server:

Reviewer: ```suggestion
To renew a certificate for a standalone server, follow these steps:
```

. From Anypoint Platform, select *Runtime Manager*.
. Click the *Servers* tab.
. Verify that the server status is `Running`.
. Click the server name.
+
image::rtm-server-standalone-name.png[Select Server]
. Click *Settings*.
. From the *Actions* menu, select *Renew Certificate*.
+
image::rtm-server-standalone-settings.png[Settings Tab]
. Select the checkbox to confirm your choice, and then click *Renew*.
+
image::rtm-server-standalone-renew.png[Confirm Renewal]
. Check *Certificate expiration date* to verify that the certificate renewed successfully.

> To renew a certificate for servers in a cluster or group:

Reviewer: ```suggestion
To renew a certificate for servers in a cluster or group, follow these steps:
```

. From Anypoint Platform, select *Runtime Manager*.
. Click the *Servers* tab.
. Verify that the server status is `Running`.
. Click the group or cluster name.
+
image::rtm-server-certificate-group-name.png[Select Group or Cluster]
. Check the radio button to select the server or servers you want to update.
. From the *Actions* menu, select *Renew Certificate*.
+
image::rtm-server-certificate-group-select-server.png[Select Server]
. Select the checkbox to confirm your choice, and then click *Renew*.
+
image::rtm-server-certificate-group-renew.png[Confirm Renewal]
. Check *Certificate expiration date* to verify that the certificate renewed successfully.

> If, after following the steps previously described, the *Renew Certificate* option in the *Actions* pull-don menu is not available, you might need to xref:installing-and-configuring-runtime-manager-agent#update-the-agent.adoc[update your *Agent* version].

Reviewer 2: typo pull-down

Author: Typo fixed! Thank you!! 😊

> If, after following the steps previously described, the *Renew Certificate* option in the *Actions* pull-down menu is not available, you might need to xref:installing-and-configuring-runtime-manager-agent#update-the-agent.adoc[update your *Agent* version].

Reviewer: ```suggestion
If the *Renew Certificate* option in the *Actions* pull-down menu is not available after performing the previous steps, you may need to xref:installing-and-configuring-runtime-manager-agent#update-the-agent.adoc[update your *Agent* version].
```

The *Server certificate renewal* feature, which enables you to renew server certificates directly from Runtime Manager, was introduced by Runtime Manager 2.8.4. This feature requires Runtime Manager Agent 1.10.0 or later.

> You can hover over the *Renew Certificate* option in the *Actions* pull-don menu, and a tooltip in the UI will tell you the minimum required Runtime Agent version.

Reviewer: ```suggestion
You can hover over the *Renew Certificate* option in the *Actions* pull-down menu, and a tooltip in the UI will indicate you the minimum required Runtime Agent version.
```
